{'relevance': '1', 'generate_bug': '0', 'comment': '<p><br></p>', 'level': '2', 'id': '53689'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p><br></p>', 'level': '3', 'id': '53689'}
{'relevance': '1', 'generate_bug': '1', 'comment': '<p>Although does not trigger the same but, it produces the following bug which is worth looking at:</p><p>The full traceback is:</p><p>Traceback (most recent call last):</p><p>&nbsp;File "/tmp/ansible_pip_payload_y4mdkro9/ansible_pip_payload.zip/ansible/modules/pip.py", line 280, in &lt;module&gt;</p><p>ModuleNotFoundError: No module named \'pkg_resources\'</p><p>fatal: [node3]: FAILED! =&gt; {</p><p>&nbsp;&nbsp;&nbsp;"changed": false,</p><p>&nbsp;&nbsp;&nbsp;"invocation": {</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"module_args": {</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"chdir": null,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"editable": false,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"executable": null,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"extra_args": null,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"name": [</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"ansible-callback-slack"</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;],</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"requirements": null,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"state": "present",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"umask": null,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"version": null,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"virtualenv": null,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"virtualenv_command": "virtualenv",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"virtualenv_python": null,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"virtualenv_site_packages": false</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;}</p><p>&nbsp;&nbsp;&nbsp;},</p><p>&nbsp;&nbsp;&nbsp;"msg": "Failed to import the required Python library (setuptools) on 2bb3030bf899\'s Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"</p><p>}</p>', 'level': '4', 'id': '53689'}
{'relevance': '2', 'generate_bug': '0', 'comment': "<p>This following issue is related with this issue: https://github.com/ansible/ansible/issues/31805 . One example playbook is: #main.yml</p><p>- hosts: development&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;become: yes&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;gather_facts: no&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;tasks:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;- name: print debug msg&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;debug:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;msg: 'Hello, World!' This ansible code tries to generate a custom callback through `custom_callback.py` file. It may attempt to generate existing error but not triggers new bug.</p>", 'level': 'level: 3', 'id': 'ID: 30597.\nAnsible Github Link: Issue Link'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p><br></p>', 'level': 'level: 4', 'id': 'ID: 30597.\nAnsible Github Link: Issue Link'}
{'relevance': '2', 'generate_bug': '0', 'comment': "<p>This following issue is related with this issue: https://github.com/ansible/ansible/issues/31805 . One example playbook is: #main.yml</p><p>- hosts: development&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;become: yes&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;gather_facts: no&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;tasks:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;- name: print debug msg&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;debug:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;msg: 'Hello, World!' This ansible code tries to generate a custom callback through `custom_callback.py` file. It may attempt to generate existing error but not triggers new bug.</p>", 'level': 'level: 3', 'id': 'ID: 30597.\nAnsible Github Link: Issue Link'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p><br></p>', 'level': 'level: 4', 'id': 'ID: 30597.\nAnsible Github Link: Issue Link'}
{'relevance': '2', 'generate_bug': '0', 'comment': "<p>This following issue is related with this issue: https://github.com/ansible/ansible/issues/31805 . One example playbook is: #main.yml</p><p>- hosts: development&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;become: yes&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;gather_facts: no&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;tasks:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;- name: print debug msg&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;debug:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;msg: 'Hello, World!' This ansible code tries to generate a custom callback through `custom_callback.py` file. It may attempt to generate existing error but not triggers new bug.</p>", 'level': 'level: 3', 'id': 'ID: 30597.\nAnsible Github Link: Issue Link'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p><br></p>', 'level': 'level: 4', 'id': 'ID: 30597.\nAnsible Github Link: Issue Link'}
{'relevance': '2', 'generate_bug': '0', 'comment': "<p>This following issue is related with this issue: https://github.com/ansible/ansible/issues/31805 . One example playbook is: #main.yml</p><p>- hosts: development&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;become: yes&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;gather_facts: no&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;tasks:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;- name: print debug msg&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;debug:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;msg: 'Hello, World!' This ansible code tries to generate a custom callback through `custom_callback.py` file. It may attempt to generate existing error but not triggers new bug.</p>", 'level': '3', 'id': '30597'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p><br></p>', 'level': '4', 'id': '30597'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>produced ansible code does not trigger any similar bug.</p>', 'level': '2', 'id': '11627'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p><br></p>', 'level': '2', 'id': '31805'}
{'relevance': '2', 'generate_bug': '0', 'comment': '<p>this produced code is similar to example code in the issue. But Once I changed the {{ callbackModule }} to "Hello World!" it did not reproduce the same error. I am guessing we need some custom callback functions to recreate this error. Ansible version: 2.4.0.0</p>', 'level': '4', 'id': '31805'}
{'relevance': '2', 'generate_bug': '0', 'comment': '<p>Very relevant to the actual cause of the issue.&nbsp;</p>', 'level': '3', 'id': '37562'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p>does not calls any callback module</p>', 'level': '4', 'id': '37562'}
{'relevance': '2', 'generate_bug': '1', 'comment': '<p>Seems like it is producing the error from the title.&nbsp;</p>', 'level': '4', 'id': '56464'}
{'relevance': '2', 'generate_bug': '1', 'comment': '<p>seems like this has introduced an error. In the issue they said that the issue is only visible for redhat. But in our environment we can see the issue in ubuntu</p>', 'level': '2', 'id': '20602'}
{'relevance': '1', 'generate_bug': '1', 'comment': '<p>seems like this has introduced an error. Need to check if the error message is valid or not.</p>', 'level': '3', 'id': '20602'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>Incomplete</p>', 'level': '3', 'id': '27676'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>Cisco device</p>', 'level': '4', 'id': '27676'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p>Works with appropriate module but does not do desired function.</p>', 'level': '2', 'id': '27926'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p>doesn’t test for appropriate error.</p>', 'level': '4', 'id': '27926'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>The module can be used to manage IGMP interface configuration settings on Cisco NX-OS devices. IGMP stands for Internet Group Management Protocol, which is a protocol that allows hosts to join and leave multicast groups on a network. Multicast groups are groups of hosts that want to receive the same data from a source.</p>', 'level': '2', 'id': '27484'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>The module can be used to manage IGMP interface configuration settings on Cisco NX-OS devices. IGMP stands for Internet Group Management Protocol, which is a protocol that allows hosts to join and leave multicast groups on a network. Multicast groups are groups of hosts that want to receive the same data from a source.</p>', 'level': '3', 'id': '27484'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>The module can be used to manage IGMP interface configuration settings on Cisco NX-OS devices. IGMP stands for Internet Group Management Protocol, which is a protocol that allows hosts to join and leave multicast groups on a network. Multicast groups are groups of hosts that want to receive the same data from a source.</p>', 'level': '4', 'id': '27484'}
{'relevance': '2', 'generate_bug': '0', 'comment': '<p>need to fix docker env for accepting sudo access to ansible. After giving the become password the output is as follows: PLAY [Test vmware_portgroup idempotecy issue] *******************************************************************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] ******************************************************************************************************************************************************************************************************</p><p>ok: [node1]</p><p>ok: [node3]</p><p>ok: [node2]</p><p><br></p><p>TASK [Create vmware_portgroup] **********************************************************************************************************************************************************************************************</p><p>fatal: [node1]: FAILED! =&gt; {"msg": "The task includes an option with an undefined variable. The error was: \'vault_password\' is undefined. \'vault_password\' is undefined\\n\\nThe error appears to be in \'/home/sibat/repoes/iac-compiler-quality/data/generated_yaml/20230513-233833/lv2/59917.yaml\': line 8, column 5, but may\\nbe elsewhere in the file depending on the exact syntax problem.\\n\\nThe offending line appears to be:\\n\\n\\n&nbsp;- name: Create vmware_portgroup\\n&nbsp;&nbsp;&nbsp;^ here\\n"}</p><p>fatal: [node2]: FAILED! =&gt; {"msg": "The task includes an option with an undefined variable. The error was: \'vault_password\' is undefined. \'vault_password\' is undefined\\n\\nThe error appears to be in \'/home/sibat/repoes/iac-compiler-quality/data/generated_yaml/20230513-233833/lv2/59917.yaml\': line 8, column 5, but may\\nbe elsewhere in the file depending on the exact syntax problem.\\n\\nThe offending line appears to be:\\n\\n\\n&nbsp;- name: Create vmware_portgroup\\n&nbsp;&nbsp;&nbsp;^ here\\n"}</p><p>fatal: [node3]: FAILED! =&gt; {"msg": "The task includes an option with an undefined variable. The error was: \'vault_password\' is undefined. \'vault_password\' is undefined\\n\\nThe error appears to be in \'/home/sibat/repoes/iac-compiler-quality/data/generated_yaml/20230513-233833/lv2/59917.yaml\': line 8, column 5, but may\\nbe elsewhere in the file depending on the exact syntax problem.\\n\\nThe offending line appears to be:\\n\\n\\n&nbsp;- name: Create vmware_portgroup\\n&nbsp;&nbsp;&nbsp;^ here\\n"}</p><p><br></p><p>PLAY RECAP ******************************************************************************************************************************************************************************************************************</p><p>node1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;&nbsp;changed=0&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;&nbsp;changed=0&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;&nbsp;changed=0&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p>', 'level': '2', 'id': '59917'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p>need to update prompt and provide the ansible config file in the prompt.</p>', 'level': '3', 'id': '59917'}
{'relevance': '2', 'generate_bug': '0', 'comment': '<p>need to update prompt and provide the ansible config file in the prompt.</p>', 'level': '4', 'id': '59917'}
{'relevance': '2', 'generate_bug': '0', 'comment': '<p>need to fix docker env for accepting sudo access to ansible. After giving the become password the output is as follows: PLAY [Test vmware_portgroup idempotecy issue] *******************************************************************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] ******************************************************************************************************************************************************************************************************</p><p>ok: [node1]</p><p>ok: [node3]</p><p>ok: [node2]</p><p><br></p><p>TASK [Create vmware_portgroup] **********************************************************************************************************************************************************************************************</p><p>fatal: [node1]: FAILED! =&gt; {"msg": "The task includes an option with an undefined variable. The error was: \'vault_password\' is undefined. \'vault_password\' is undefined\\n\\nThe error appears to be in \'/home/sibat/repoes/iac-compiler-quality/data/generated_yaml/20230513-233833/lv2/59917.yaml\': line 8, column 5, but may\\nbe elsewhere in the file depending on the exact syntax problem.\\n\\nThe offending line appears to be:\\n\\n\\n&nbsp;- name: Create vmware_portgroup\\n&nbsp;&nbsp;&nbsp;^ here\\n"}</p><p>fatal: [node2]: FAILED! =&gt; {"msg": "The task includes an option with an undefined variable. The error was: \'vault_password\' is undefined. \'vault_password\' is undefined\\n\\nThe error appears to be in \'/home/sibat/repoes/iac-compiler-quality/data/generated_yaml/20230513-233833/lv2/59917.yaml\': line 8, column 5, but may\\nbe elsewhere in the file depending on the exact syntax problem.\\n\\nThe offending line appears to be:\\n\\n\\n&nbsp;- name: Create vmware_portgroup\\n&nbsp;&nbsp;&nbsp;^ here\\n"}</p><p>fatal: [node3]: FAILED! =&gt; {"msg": "The task includes an option with an undefined variable. The error was: \'vault_password\' is undefined. \'vault_password\' is undefined\\n\\nThe error appears to be in \'/home/sibat/repoes/iac-compiler-quality/data/generated_yaml/20230513-233833/lv2/59917.yaml\': line 8, column 5, but may\\nbe elsewhere in the file depending on the exact syntax problem.\\n\\nThe offending line appears to be:\\n\\n\\n&nbsp;- name: Create vmware_portgroup\\n&nbsp;&nbsp;&nbsp;^ here\\n"}</p><p><br></p><p>PLAY RECAP ******************************************************************************************************************************************************************************************************************</p><p>node1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;&nbsp;changed=0&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;&nbsp;changed=0&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;&nbsp;changed=0&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p>', 'level': '2', 'id': '59917'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p>need to update prompt and provide the ansible config file in the prompt.</p>', 'level': '3', 'id': '59917'}
{'relevance': '2', 'generate_bug': '0', 'comment': '<p>need to update prompt and provide the ansible config file in the prompt.</p>', 'level': '4', 'id': '59917'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>we are not working with azure</p>', 'level': '2', 'id': '45330'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>we are not working with azure</p>', 'level': '3', 'id': '45330'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>we are not working with azure</p>', 'level': '4', 'id': '45330'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p>produces another error which is worth looking at</p>', 'level': '2', 'id': '22459'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p>Need to produce yaml with those variable values. Current output after sudo pass:ansible-playbook data/generated_yaml/20230513-233833/lv3/22459.yaml -i confs/inventory --ask-become-pass</p><p>BECOME password:</p><p><br></p><p>PLAY [Exposing Unicode idempotent issues in eos_banner module] **************************************************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] ******************************************************************************************************************************************************************************************************</p><p>ok: [node1]</p><p><br></p><p>TASK [Install required packages] ********************************************************************************************************************************************************************************************</p><p>failed: [node1] (item=package1) =&gt; {"ansible_loop_var": "item", "changed": false, "item": "package1", "msg": "No package matching \'package1\' is available"}</p><p>failed: [node1] (item=package2) =&gt; {"ansible_loop_var": "item", "changed": false, "item": "package2", "msg": "No package matching \'package2\' is available"}</p><p><br></p><p>PLAY RECAP ******************************************************************************************************************************************************************************************************************</p><p>node1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;&nbsp;changed=0&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p>', 'level': '3', 'id': '22459'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>OS / ENVIRONMENT</p><p>Running Ansible from Centos 7.2 and testing ios_config module on a Cisco IOS device running version.</p>', 'level': '3', 'id': '40064'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>OS / ENVIRONMENT</p><p>Running Ansible from Centos 7.2 and testing ios_config module on a Cisco IOS device running version.</p>', 'level': '4', 'id': '40064'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>OS / ENVIRONMENT</p><p>N/A</p><p>Cisco IOS Software, IOS-XE Software, Catalyst L3 Switch Software (CAT3K_CAA-UNIVERSALK9-M), Version 03.03.05SE RELEASE SOFTWARE (fc1)</p><p>Catalyst 3850</p>', 'level': '2', 'id': '38301'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>OS / ENVIRONMENT</p><p>N/A</p><p>Cisco IOS Software, IOS-XE Software, Catalyst L3 Switch Software (CAT3K_CAA-UNIVERSALK9-M), Version 03.03.05SE RELEASE SOFTWARE (fc1)</p><p>Catalyst 3850</p>', 'level': '4', 'id': '38301'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>Kubernate is an orchastrator which is out of our testing scope</p>', 'level': '3', 'id': '31602'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>Kubernate is an orchastrator which is out of our testing scope</p>', 'level': '4', 'id': '31602'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>OS / ENVIRONMENT</p><p>CentOS Linux release 7.2.1511 (Core)</p>', 'level': '3', 'id': '29429'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>OS / ENVIRONMENT</p><p>CentOS Linux release 7.2.1511 (Core)</p>', 'level': '4', 'id': '29429'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>The nxos_config module allows users to send arbitrary configuration commands to Cisco Nexus devices running NX-OS software.</p>', 'level': '2', 'id': '37022'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>The nxos_config module allows users to send arbitrary configuration commands to Cisco Nexus devices running NX-OS software.</p>', 'level': '3', 'id': '37022'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>The nxos_config module allows users to send arbitrary configuration commands to Cisco Nexus devices running NX-OS software.</p>', 'level': '3', 'id': '36801'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>The nxos_config module allows users to send arbitrary configuration commands to Cisco Nexus devices running NX-OS software.</p>', 'level': '3', 'id': '30439'}
{'relevance': '2', 'generate_bug': '1', 'comment': '<p>Need to check if port id is supposed to be inside os_server</p>', 'level': '3', 'id': '54652'}
{'relevance': '2', 'generate_bug': '0', 'comment': '<p>potential bug</p>', 'level': '2', 'id': '49437'}
{'relevance': '2', 'generate_bug': '1', 'comment': '<p>Potential bug</p>', 'level': '3', 'id': '49437'}
{'relevance': '2', 'generate_bug': '1', 'comment': '<p>Updated output : PLAY [Test User module idempotency] *****************************************************************************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] ******************************************************************************************************************************************************************************************************</p><p>ok: [node1]</p><p>ok: [node2]</p><p>ok: [node3]</p><p><br></p><p>TASK [Ensure the user does not exist] ***************************************************************************************************************************************************************************************</p><p>ok: [node1]</p><p>ok: [node3]</p><p>ok: [node2]</p><p><br></p><p>TASK [Ensure the user exists and has a UID of 1001] *************************************************************************************************************************************************************************</p><p>changed: [node1]</p><p>changed: [node2]</p><p>changed: [node3]</p><p><br></p><p>TASK [Ensure the user exists but with unconventional syntax] ****************************************************************************************************************************************************************</p><p>fatal: [node1]: FAILED! =&gt; {"changed": false, "msg": "useradd: UID 1001 is not unique\\n", "name": "te\'stus\'&amp;#$%er", "rc": 4}</p><p>fatal: [node2]: FAILED! =&gt; {"changed": false, "msg": "useradd: UID 1001 is not unique\\n", "name": "te\'stus\'&amp;#$%er", "rc": 4}</p><p>fatal: [node3]: FAILED! =&gt; {"changed": false, "msg": "useradd: UID 1001 is not unique\\n", "name": "te\'stus\'&amp;#$%er", "rc": 4}</p><p><br></p><p>PLAY RECAP ******************************************************************************************************************************************************************************************************************</p><p>node1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=3&nbsp;&nbsp;&nbsp;changed=1&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=3&nbsp;&nbsp;&nbsp;changed=1&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=3&nbsp;&nbsp;&nbsp;changed=1&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p>', 'level': '2', 'id': '35291'}
{'relevance': '2', 'generate_bug': '0', 'comment': '<p>Updated output: PLAY [Check if user exists] *************************************************************************************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] ******************************************************************************************************************************************************************************************************</p><p>ok: [node1]</p><p><br></p><p>TASK [Check if non-existing user exists] ************************************************************************************************************************************************************************************</p><p>changed: [node1]</p><p><br></p><p>TASK [Debug output] *********************************************************************************************************************************************************************************************************</p><p>ok: [node1] =&gt; {</p><p>&nbsp;&nbsp;&nbsp;"check_user": {</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"changed": true,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"comment": "",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"create_home": true,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"failed": false,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"group": 1001,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"home": "/home/test_user",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"name": "test_user",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"shell": "/bin/sh",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"state": "present",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"system": false,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"uid": 1001</p><p>&nbsp;&nbsp;&nbsp;}</p><p>}</p><p><br></p><p>PLAY RECAP ******************************************************************************************************************************************************************************************************************</p><p>node1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=3&nbsp;&nbsp;&nbsp;changed=1&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=0&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p>', 'level': '3', 'id': '35291'}
{'relevance': '2', 'generate_bug': '1', 'comment': "<p>Ran 2 times and both time creates user. PLAY [Test User Module] *****************************************************************************************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] ******************************************************************************************************************************************************************************************************</p><p>ok: [node1]</p><p>ok: [node3]</p><p>ok: [node2]</p><p><br></p><p>TASK [Create user] **********************************************************************************************************************************************************************************************************</p><p>[WARNING]: The input password appears not to have been hashed. The 'password' argument must be encrypted for this module to work properly.</p><p>changed: [node3]</p><p>changed: [node2]</p><p>changed: [node1]</p><p><br></p><p>TASK [Modify user] **********************************************************************************************************************************************************************************************************</p><p>changed: [node3]</p><p>changed: [node2]</p><p>changed: [node1]</p><p><br></p><p>TASK [Delete user] **********************************************************************************************************************************************************************************************************</p><p>changed: [node1]</p><p>changed: [node3]</p><p>changed: [node2]</p><p><br></p><p>PLAY RECAP ******************************************************************************************************************************************************************************************************************</p><p>node1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=4&nbsp;&nbsp;&nbsp;changed=3&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=0&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=4&nbsp;&nbsp;&nbsp;changed=3&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=0&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=4&nbsp;&nbsp;&nbsp;changed=3&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=0&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p><br></p><p>(fuzz) sibat@nlp:~/repoes/iac-compiler-quality$ ansible-playbook data/generated_yaml/20230513-233833/lv4/35291.yaml -i confs/inventory --become-password-file files/docker_scripts/ssh/password.txt</p><p><br></p><p>PLAY [Test User Module] *****************************************************************************************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] ******************************************************************************************************************************************************************************************************</p><p>ok: [node3]</p><p>ok: [node2]</p><p>ok: [node1]</p><p><br></p><p>TASK [Create user] **********************************************************************************************************************************************************************************************************</p><p>[WARNING]: The input password appears not to have been hashed. The 'password' argument must be encrypted for this module to work properly.</p><p>changed: [node3]</p><p>changed: [node2]</p><p>changed: [node1]</p><p><br></p><p>TASK [Modify user] **********************************************************************************************************************************************************************************************************</p><p>changed: [node1]</p><p>changed: [node2]</p><p>changed: [node3]</p><p><br></p><p>TASK [Delete user] **********************************************************************************************************************************************************************************************************</p><p>changed: [node1]</p><p>changed: [node3]</p><p>changed: [node2]</p><p><br></p><p>PLAY RECAP ******************************************************************************************************************************************************************************************************************</p><p>node1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=4&nbsp;&nbsp;&nbsp;changed=3&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=0&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=4&nbsp;&nbsp;&nbsp;changed=3&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=0&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=4&nbsp;&nbsp;&nbsp;changed=3&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=0&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p>", 'level': '4', 'id': '35291'}
{'relevance': '2', 'generate_bug': '1', 'comment': '<p>Need to check if networks_info actually should be in the parameters of vmware_guest.</p>', 'level': '3', 'id': '34105'}
{'relevance': '2', 'generate_bug': '0', 'comment': '<p>Variables need to be handled in the prompt.</p>', 'level': '4', 'id': '34105'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p>variables needs to be handled</p>', 'level': '2', 'id': '24511'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p>need to handle variables</p>', 'level': '3', 'id': '24511'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p>Replaced hosts: vmware to hosts: all.[WARNING]: Could not match supplied host pattern, ignoring: vmware</p><p><br></p><p>PLAY [vmware] ***************************************************************************************************************************************************************************************************************</p><p>skipping: no hosts matched</p><p><br></p><p>PLAY RECAP ******************************************************************************************************************************************************************************************************************</p><p><br></p><p>(fuzz) sibat@nlp:~/repoes/iac-compiler-quality$ vim data/generated_yaml/20230513-233833/lv4/24511.yaml</p><p>(fuzz) sibat@nlp:~/repoes/iac-compiler-quality$ ansible-playbook data/generated_yaml/20230513-233833/lv4/24511.yaml -i confs/inventory --become-password-file files/docker_scripts/ssh/password.txt</p><p><br></p><p>PLAY [all] ******************************************************************************************************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] ******************************************************************************************************************************************************************************************************</p><p>ok: [node1]</p><p>ok: [node3]</p><p>ok: [node2]</p><p><br></p><p>TASK [Take a snapshot of the VM] ********************************************************************************************************************************************************************************************</p><p>fatal: [node3]: FAILED! =&gt; {"changed": false, "msg": "missing required arguments: datacenter"}</p><p>fatal: [node1]: FAILED! =&gt; {"changed": false, "msg": "missing required arguments: datacenter"}</p><p>fatal: [node2]: FAILED! =&gt; {"changed": false, "msg": "missing required arguments: datacenter"}</p><p><br></p><p>PLAY RECAP ******************************************************************************************************************************************************************************************************************</p><p>node1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;&nbsp;changed=0&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;&nbsp;changed=0&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;&nbsp;changed=0&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p>', 'level': '4', 'id': '24511'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>Win domain</p>', 'level': '2', 'id': '47352'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>Added sudo password and updated output: PLAY [Replay the Ansible win_domain Module\'s Idempotency Issue] *************************************************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] ******************************************************************************************************************************************************************************************************</p><p>ok: [node3]</p><p>ok: [node2]</p><p>ok: [node1]</p><p><br></p><p>TASK [Join workstation to domain] *******************************************************************************************************************************************************************************************</p><p>fatal: [node1]: FAILED! =&gt; {"changed": false, "module_stderr": "Shared connection to 10.1.1.1 closed.\\r\\n", "module_stdout": "\\r\\n/bin/sh: 1: powershell: not found\\r\\n", "msg": "The module failed to execute correctly, you probably need to set the interpreter.\\nSee stdout/stderr for the exact error", "rc": 127}</p><p>fatal: [node2]: FAILED! =&gt; {"changed": false, "module_stderr": "Shared connection to 10.1.1.2 closed.\\r\\n", "module_stdout": "\\r\\n/bin/sh: 1: powershell: not found\\r\\n", "msg": "The module failed to execute correctly, you probably need to set the interpreter.\\nSee stdout/stderr for the exact error", "rc": 127}</p><p>fatal: [node3]: FAILED! =&gt; {"changed": false, "module_stderr": "Shared connection to 10.1.1.3 closed.\\r\\n", "module_stdout": "\\r\\n/bin/sh: 1: powershell: not found\\r\\n", "msg": "The module failed to execute correctly, you probably need to set the interpreter.\\nSee stdout/stderr for the exact error", "rc": 127}</p><p><br></p><p>PLAY RECAP ******************************************************************************************************************************************************************************************************************</p><p>node1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;&nbsp;changed=0&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;&nbsp;changed=0&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;&nbsp;changed=0&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p>', 'level': '3', 'id': '47352'}
{'relevance': '0', 'generate_bug': '0', 'comment': "<p>OS / ENVIRONMENT</p><p>Windows 10. NB: error is same as previous issue's generated ansible</p>", 'level': '2', 'id': '20283'}
{'relevance': '0', 'generate_bug': '0', 'comment': "<p>OS / ENVIRONMENT</p><p>Windows 10. NB: error is same as previous issue's generated ansible</p>", 'level': '3', 'id': '20283'}
{'relevance': '0', 'generate_bug': '0', 'comment': "<p>OS / ENVIRONMENT</p><p>Windows 10. NB: error is same as previous issue's generated ansible</p>", 'level': '4', 'id': '20283'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>ENV: Windows</p>', 'level': '3', 'id': '28758'}
{'relevance': '1', 'generate_bug': '1', 'comment': '<p>produces the expected error</p>', 'level': '2', 'id': '68509'}
{'relevance': '2', 'generate_bug': '1', 'comment': '<p>I think this is another bug.</p>', 'level': '4', 'id': '68509'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>None</p>', 'level': '2', 'id': '43015'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>None</p>', 'level': '3', 'id': '43015'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>None</p>', 'level': '4', 'id': '75817'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>None</p>', 'level': '2', 'id': '13292'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>None</p>', 'level': '3', 'id': '13292'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>None</p>', 'level': '4', 'id': '13292'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>None</p>', 'level': '2', 'id': '13292'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>None</p>', 'level': '3', 'id': '13292'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>None</p>', 'level': '4', 'id': '13292'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p>None</p>', 'level': '2', 'id': '15523'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p>None</p>', 'level': '3', 'id': '15523'}
{'relevance': '2', 'generate_bug': '1', 'comment': '<p>Double check needed</p>', 'level': '2', 'id': '72518'}
{'relevance': '2', 'generate_bug': '1', 'comment': '<p>Ran with sudo password. Gives the following output:</p><p>PLAY [Test playbook for uncovering Ansible compiler bugs] *******************************************************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] ******************************************************************************************************************************************************************************************************</p><p>ok: [node2]</p><p>ok: [node1]</p><p>ok: [node3]</p><p><br></p><p>TASK [Create a new directory on the remote host.] ***************************************************************************************************************************************************************************</p><p>changed: [node1]</p><p>changed: [node3]</p><p>changed: [node2]</p><p><br></p><p>TASK [Add a custom fact with an incorrect character.] ***********************************************************************************************************************************************************************</p><p>fatal: [node3]: FAILED! =&gt; {"changed": false, "checksum": "cf3101838ddb05768f78f127d6e200579cf9ced7", "msg": "Destination directory /etc/ansible/facts.d does not exist"}</p><p>fatal: [node2]: FAILED! =&gt; {"changed": false, "checksum": "cf3101838ddb05768f78f127d6e200579cf9ced7", "msg": "Destination directory /etc/ansible/facts.d does not exist"}</p><p>fatal: [node1]: FAILED! =&gt; {"changed": false, "checksum": "cf3101838ddb05768f78f127d6e200579cf9ced7", "msg": "Destination directory /etc/ansible/facts.d does not exist"}</p><p><br></p><p>PLAY RECAP ******************************************************************************************************************************************************************************************************************</p><p>node1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;&nbsp;changed=1&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;&nbsp;changed=1&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;&nbsp;changed=1&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p>', 'level': '4', 'id': '72518'}
{'relevance': '2', 'generate_bug': '1', 'comment': '<p>Double check needed</p>', 'level': '2', 'id': '72518'}
{'relevance': '2', 'generate_bug': '1', 'comment': '<p>Ran with sudo password. Gives the following output:</p><p>PLAY [Test playbook for uncovering Ansible compiler bugs] *******************************************************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] ******************************************************************************************************************************************************************************************************</p><p>ok: [node2]</p><p>ok: [node1]</p><p>ok: [node3]</p><p><br></p><p>TASK [Create a new directory on the remote host.] ***************************************************************************************************************************************************************************</p><p>changed: [node1]</p><p>changed: [node3]</p><p>changed: [node2]</p><p><br></p><p>TASK [Add a custom fact with an incorrect character.] ***********************************************************************************************************************************************************************</p><p>fatal: [node3]: FAILED! =&gt; {"changed": false, "checksum": "cf3101838ddb05768f78f127d6e200579cf9ced7", "msg": "Destination directory /etc/ansible/facts.d does not exist"}</p><p>fatal: [node2]: FAILED! =&gt; {"changed": false, "checksum": "cf3101838ddb05768f78f127d6e200579cf9ced7", "msg": "Destination directory /etc/ansible/facts.d does not exist"}</p><p>fatal: [node1]: FAILED! =&gt; {"changed": false, "checksum": "cf3101838ddb05768f78f127d6e200579cf9ced7", "msg": "Destination directory /etc/ansible/facts.d does not exist"}</p><p><br></p><p>PLAY RECAP ******************************************************************************************************************************************************************************************************************</p><p>node1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;&nbsp;changed=1&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;&nbsp;changed=1&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p><p>node3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;&nbsp;changed=1&nbsp;&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;&nbsp;failed=1&nbsp;&nbsp;&nbsp;skipped=0&nbsp;&nbsp;&nbsp;rescued=0&nbsp;&nbsp;&nbsp;ignored=0</p>', 'level': '4', 'id': '72518'}
{'relevance': '2', 'generate_bug': '0', 'comment': '<p>Different errors generated</p>', 'level': '3', 'id': '16760'}
{'relevance': '2', 'generate_bug': '0', 'comment': '<p>PLAY [Recreate Ansible compiler bug] ********************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] **********************************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: Platform linux on host alpine1 is using the discovered Python interpreter at /usr/bin/python3.11, but future installation of another Python interpreter</p><p>could change the meaning of that path. See https://docs.ansible.com/ansible-core/2.14/reference_appendices/interpreter_discovery.html for more information.</p><p>ok: [alpine1]</p><p>ok: [ubuntu1]</p><p>ok: [redhat1]</p><p>ok: [centos1]</p><p><br></p><p>TASK [Run ansible-galaxy command with incorrect attribute] **********************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>fatal: [ubuntu1]: FAILED! =&gt; {"changed": true, "cmd": "ansible-galaxy info --offline", "delta": "0:00:00.001211", "end": "2023-05-25 06:31:10.628609", "msg": "non-zero return code", "rc": 127, "start": "2023-05-25 06:31:10.627398", "stderr": "/bin/sh: 1: ansible-galaxy: not found", "stderr_lines": ["/bin/sh: 1: ansible-galaxy: not found"], "stdout": "", "stdout_lines": []}</p><p>fatal: [alpine1]: FAILED! =&gt; {"changed": true, "cmd": "ansible-galaxy info --offline", "delta": "0:00:00.002074", "end": "2023-05-25 06:31:10.687797", "msg": "non-zero return code", "rc": 127, "start": "2023-05-25 06:31:10.685723", "stderr": "/bin/sh: ansible-galaxy: not found", "stderr_lines": ["/bin/sh: ansible-galaxy: not found"], "stdout": "", "stdout_lines": []}</p><p>fatal: [centos1]: FAILED! =&gt; {"changed": true, "cmd": "ansible-galaxy info --offline", "delta": "0:00:00.002146", "end": "2023-05-25 06:31:10.691514", "msg": "non-zero return code", "rc": 127, "start": "2023-05-25 06:31:10.689368", "stderr": "/bin/sh: ansible-galaxy: command not found", "stderr_lines": ["/bin/sh: ansible-galaxy: command not found"], "stdout": "", "stdout_lines": []}</p><p>fatal: [redhat1]: FAILED! =&gt; {"changed": true, "cmd": "ansible-galaxy info --offline", "delta": "0:00:00.002472", "end": "2023-05-25 06:31:10.718586", "msg": "non-zero return code", "rc": 127, "start": "2023-05-25 06:31:10.716114", "stderr": "/bin/sh: ansible-galaxy: command not found", "stderr_lines": ["/bin/sh: ansible-galaxy: command not found"], "stdout": "", "stdout_lines": []}</p><p><br></p><p>PLAY RECAP **********************************************************************************************************************************************************</p><p>alpine1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>centos1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>redhat1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>ubuntu1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p><br></p><h2><strong>Discussion:</strong></h2><p><br></p><p>there\'s a mistake in this task. The <code style="color: var(--darkreader-text--tw-prose-code); --darkreader-inline-color: var(--darkreader-text--darkreader-text--tw-prose-code);" data-darkreader-inline-color="">ansible-galaxy info --offline</code> command requires a role name to display information about, but none is provided here. Therefore, this command will fail with an error.</p><p>You\'d typically use this command like so: <code style="color: var(--darkreader-text--tw-prose-code); --darkreader-inline-color: var(--darkreader-text--darkreader-text--tw-prose-code);" data-darkreader-inline-color="">ansible-galaxy info --offline &lt;role_name&gt;</code>, replacing <code style="color: var(--darkreader-text--tw-prose-code); --darkreader-inline-color: var(--darkreader-text--darkreader-text--tw-prose-code);" data-darkreader-inline-color="">&lt;role_name&gt;</code> with the name of the installed role you want information about.</p><p>Also, Ansible and ansible-galaxy commands are usually only installed and run on the Ansible control node, not on the target hosts. It\'s rare to see a playbook that runs <code style="color: var(--darkreader-text--tw-prose-code); --darkreader-inline-color: var(--darkreader-text--darkreader-text--tw-prose-code);" data-darkreader-inline-color="">ansible-galaxy</code> commands on the target hosts.</p><p><br></p>', 'level': '3', 'id': '16760'}
{'relevance': '2', 'generate_bug': '1', 'comment': '<p>PLAY [Create directory with unconventional permissions] *************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] **********************************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>ok: [ubuntu1]</p><p>[WARNING]: Platform linux on host alpine1 is using the discovered Python interpreter at /usr/bin/python3.11, but future installation of another Python interpreter</p><p>could change the meaning of that path. See https://docs.ansible.com/ansible-core/2.14/reference_appendices/interpreter_discovery.html for more information.</p><p>ok: [alpine1]</p><p>ok: [redhat1]</p><p>ok: [centos1]</p><p><br></p><p>TASK [Create directory with permissions 0621] ***********************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>ok: [ubuntu1]</p><p>ok: [alpine1]</p><p>ok: [centos1]</p><p>ok: [redhat1]</p><p><br></p><p>TASK [Create directory with permissions \'0.5\'] **********************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>fatal: [ubuntu1]: FAILED! =&gt; {"changed": false, "details": "bad symbolic permission for mode: 0.5", "gid": 0, "group": "root", "mode": "0755", "msg": "mode must be in octal or symbolic form", "owner": "root", "path": "/tmp/testdir2", "size": 4096, "state": "directory", "uid": 0}</p><p>fatal: [alpine1]: FAILED! =&gt; {"changed": false, "details": "bad symbolic permission for mode: 0.5", "gid": 0, "group": "root", "mode": "0755", "msg": "mode must be in octal or symbolic form", "owner": "root", "path": "/tmp/testdir2", "size": 4096, "state": "directory", "uid": 0}</p><p>fatal: [centos1]: FAILED! =&gt; {"changed": false, "details": "bad symbolic permission for mode: 0.5", "gid": 0, "group": "root", "mode": "0755", "msg": "mode must be in octal or symbolic form", "owner": "root", "path": "/tmp/testdir2", "size": 4096, "state": "directory", "uid": 0}</p><p>fatal: [redhat1]: FAILED! =&gt; {"changed": false, "details": "bad symbolic permission for mode: 0.5", "gid": 0, "group": "root", "mode": "0755", "msg": "mode must be in octal or symbolic form", "owner": "root", "path": "/tmp/testdir2", "size": 4096, "state": "directory", "uid": 0}</p><p><br></p><p>PLAY RECAP **********************************************************************************************************************************************************</p><p>alpine1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>centos1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>redhat1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>ubuntu1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p><br></p><p><br></p><p><br></p><p><br></p><p>Ran the following code where the mode is in integer form</p><p><br></p><pre class="ql-syntax" spellcheck="false">- become: true\n  hosts: all\n  name: Create directory with unconventional permissions\n  tasks:\n  - file:\n      mode: 0621\n      path: /tmp/testdir\n      state: directory\n    name: Create directory with permissions 0621\n  - file:\n      mode: 0.5\n      path: /tmp/testdir2\n      state: directory\n    name: Create directory with permissions \'0.5\'\n  - file:\n      mode: 2002\n      path: /tmp/testdir3\n      state: directory\n    name: Create directory with permissions 2002\n</pre><p><br></p><p><br></p><p>PLAY [Create directory with unconventional permissions] *************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] **********************************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: Platform linux on host alpine1 is using the discovered Python interpreter at /usr/bin/python3.11, but future installation of another Python interpreter</p><p>could change the meaning of that path. See https://docs.ansible.com/ansible-core/2.14/reference_appendices/interpreter_discovery.html for more information.</p><p>ok: [alpine1]</p><p>ok: [ubuntu1]</p><p>ok: [redhat1]</p><p>ok: [centos1]</p><p><br></p><p>TASK [Create directory with permissions 0621] ***********************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>ok: [ubuntu1]</p><p>ok: [alpine1]</p><p>ok: [centos1]</p><p>ok: [redhat1]</p><p><br></p><p>TASK [Create directory with permissions \'0.5\'] **********************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>fatal: [ubuntu1]: FAILED! =&gt; {"changed": false, "details": "\'float\' object has no attribute \'split\'", "gid": 0, "group": "root", "mode": "0755", "msg": "mode must be in octal or symbolic form", "owner": "root", "path": "/tmp/testdir2", "size": 4096, "state": "directory", "uid": 0}</p><p>fatal: [alpine1]: FAILED! =&gt; {"changed": false, "details": "\'float\' object has no attribute \'split\'", "gid": 0, "group": "root", "mode": "0755", "msg": "mode must be in octal or symbolic form", "owner": "root", "path": "/tmp/testdir2", "size": 4096, "state": "directory", "uid": 0}</p><p>fatal: [centos1]: FAILED! =&gt; {"changed": false, "details": "\'float\' object has no attribute \'split\'", "gid": 0, "group": "root", "mode": "0755", "msg": "mode must be in octal or symbolic form", "owner": "root", "path": "/tmp/testdir2", "size": 4096, "state": "directory", "uid": 0}</p><p>fatal: [redhat1]: FAILED! =&gt; {"changed": false, "details": "\'float\' object has no attribute \'split\'", "gid": 0, "group": "root", "mode": "0755", "msg": "mode must be in octal or symbolic form", "owner": "root", "path": "/tmp/testdir2", "size": 4096, "state": "directory", "uid": 0}</p><p><br></p><p>PLAY RECAP **********************************************************************************************************************************************************</p><p>alpine1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>centos1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>redhat1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>ubuntu1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p><br></p><p><br></p><p>As we can see the mode must be octal which is synonyms with the issue. </p>', 'level': '2', 'id': '42563'}
{'relevance': '2', 'generate_bug': '0', 'comment': '<p>Ran with sudo access</p><p><br></p><p>PLAY [Test playbook for bad symbolic permission errors] *************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] **********************************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: Platform linux on host alpine1 is using the discovered Python interpreter at /usr/bin/python3.11, but future installation of another Python interpreter</p><p>could change the meaning of that path. See https://docs.ansible.com/ansible-core/2.14/reference_appendices/interpreter_discovery.html for more information.</p><p>ok: [alpine1]</p><p>ok: [ubuntu1]</p><p>ok: [centos1]</p><p>ok: [redhat1]</p><p><br></p><p>TASK [Create directory with incorrect mode] *************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>changed: [ubuntu1]</p><p>changed: [alpine1]</p><p>changed: [redhat1]</p><p>changed: [centos1]</p><p><br></p><p>TASK [Create directory with incorrect symbolic permission] **********************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>fatal: [ubuntu1]: FAILED! =&gt; {"changed": false, "msg": "Unsupported parameters for (file) module: symbolic_mode. Supported parameters include: _diff_peek, _original_basename, access_time, access_time_format, attributes, follow, force, group, mode, modification_time, modification_time_format, owner, path, recurse, selevel, serole, setype, seuser, src, state, unsafe_writes (attr, dest, name)."}</p><p>fatal: [alpine1]: FAILED! =&gt; {"changed": false, "msg": "Unsupported parameters for (file) module: symbolic_mode. Supported parameters include: _diff_peek, _original_basename, access_time, access_time_format, attributes, follow, force, group, mode, modification_time, modification_time_format, owner, path, recurse, selevel, serole, setype, seuser, src, state, unsafe_writes (attr, dest, name)."}</p><p>fatal: [centos1]: FAILED! =&gt; {"changed": false, "msg": "Unsupported parameters for (file) module: symbolic_mode. Supported parameters include: _diff_peek, _original_basename, access_time, access_time_format, attributes, follow, force, group, mode, modification_time, modification_time_format, owner, path, recurse, selevel, serole, setype, seuser, src, state, unsafe_writes (attr, dest, name)."}</p><p>fatal: [redhat1]: FAILED! =&gt; {"changed": false, "msg": "Unsupported parameters for (file) module: symbolic_mode. Supported parameters include: _diff_peek, _original_basename, access_time, access_time_format, attributes, follow, force, group, mode, modification_time, modification_time_format, owner, path, recurse, selevel, serole, setype, seuser, src, state, unsafe_writes (attr, dest, name)."}</p><p><br></p><p>PLAY RECAP **********************************************************************************************************************************************************</p><p>alpine1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=1&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>centos1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=1&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>redhat1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=1&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>ubuntu1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=1&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p><br></p><p><br></p><p><span style="background-color: rgb(44, 48, 50); color: rgb(222, 219, 214); --darkreader-inline-bgcolor: #232628; --darkreader-inline-color: #d1cdc7;" data-darkreader-inline-bgcolor="" data-darkreader-inline-color="">symbolic_mode is not a valid parameter hence its not a valid bug.</span></p>', 'level': '4', 'id': '42563'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>we do not have a database server in our setup</p>', 'level': '2', 'id': '60043'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>we do not have a database server in our setup</p>', 'level': '3', 'id': '60043'}
{'relevance': '2', 'generate_bug': '1', 'comment': '<p>PLAY [Create directory with unconventional permissions] *************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] **********************************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: Platform linux on host alpine1 is using the discovered Python interpreter at /usr/bin/python3.11, but future installation of another Python interpreter</p><p>could change the meaning of that path. See https://docs.ansible.com/ansible-core/2.14/reference_appendices/interpreter_discovery.html for more information.</p><p>ok: [alpine1]</p><p>ok: [ubuntu1]</p><p>ok: [redhat1]</p><p>ok: [centos1]</p><p><br></p><p>TASK [Create directory with permissions 0621] ***********************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>ok: [ubuntu1]</p><p>ok: [alpine1]</p><p>ok: [centos1]</p><p>ok: [redhat1]</p><p><br></p><p>TASK [Create directory with permissions \'0.5\'] **********************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>fatal: [ubuntu1]: FAILED! =&gt; {"changed": false, "details": "\'float\' object has no attribute \'split\'", "gid": 0, "group": "root", "mode": "0755", "msg": "mode must be in octal or symbolic form", "owner": "root", "path": "/tmp/testdir2", "size": 4096, "state": "directory", "uid": 0}</p><p>fatal: [alpine1]: FAILED! =&gt; {"changed": false, "details": "\'float\' object has no attribute \'split\'", "gid": 0, "group": "root", "mode": "0755", "msg": "mode must be in octal or symbolic form", "owner": "root", "path": "/tmp/testdir2", "size": 4096, "state": "directory", "uid": 0}</p><p>fatal: [centos1]: FAILED! =&gt; {"changed": false, "details": "\'float\' object has no attribute \'split\'", "gid": 0, "group": "root", "mode": "0755", "msg": "mode must be in octal or symbolic form", "owner": "root", "path": "/tmp/testdir2", "size": 4096, "state": "directory", "uid": 0}</p><p>fatal: [redhat1]: FAILED! =&gt; {"changed": false, "details": "\'float\' object has no attribute \'split\'", "gid": 0, "group": "root", "mode": "0755", "msg": "mode must be in octal or symbolic form", "owner": "root", "path": "/tmp/testdir2", "size": 4096, "state": "directory", "uid": 0}</p><p><br></p><p>PLAY RECAP **********************************************************************************************************************************************************</p><p>alpine1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>centos1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>redhat1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>ubuntu1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p><br></p><p><br></p><p><br></p><p>As we can see the mode must be octal which is synonyms with the issue.</p>', 'level': '2', 'id': '42563'}
{'relevance': '2', 'generate_bug': '0', 'comment': '<p>Ran with sudo access</p><p><br></p><p>PLAY [Test playbook for bad symbolic permission errors] *************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] **********************************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: Platform linux on host alpine1 is using the discovered Python interpreter at /usr/bin/python3.11, but future installation of another Python interpreter</p><p>could change the meaning of that path. See https://docs.ansible.com/ansible-core/2.14/reference_appendices/interpreter_discovery.html for more information.</p><p>ok: [alpine1]</p><p>ok: [ubuntu1]</p><p>ok: [centos1]</p><p>ok: [redhat1]</p><p><br></p><p>TASK [Create directory with incorrect mode] *************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>changed: [ubuntu1]</p><p>changed: [alpine1]</p><p>changed: [redhat1]</p><p>changed: [centos1]</p><p><br></p><p>TASK [Create directory with incorrect symbolic permission] **********************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>fatal: [ubuntu1]: FAILED! =&gt; {"changed": false, "msg": "Unsupported parameters for (file) module: symbolic_mode. Supported parameters include: _diff_peek, _original_basename, access_time, access_time_format, attributes, follow, force, group, mode, modification_time, modification_time_format, owner, path, recurse, selevel, serole, setype, seuser, src, state, unsafe_writes (attr, dest, name)."}</p><p>fatal: [alpine1]: FAILED! =&gt; {"changed": false, "msg": "Unsupported parameters for (file) module: symbolic_mode. Supported parameters include: _diff_peek, _original_basename, access_time, access_time_format, attributes, follow, force, group, mode, modification_time, modification_time_format, owner, path, recurse, selevel, serole, setype, seuser, src, state, unsafe_writes (attr, dest, name)."}</p><p>fatal: [centos1]: FAILED! =&gt; {"changed": false, "msg": "Unsupported parameters for (file) module: symbolic_mode. Supported parameters include: _diff_peek, _original_basename, access_time, access_time_format, attributes, follow, force, group, mode, modification_time, modification_time_format, owner, path, recurse, selevel, serole, setype, seuser, src, state, unsafe_writes (attr, dest, name)."}</p><p>fatal: [redhat1]: FAILED! =&gt; {"changed": false, "msg": "Unsupported parameters for (file) module: symbolic_mode. Supported parameters include: _diff_peek, _original_basename, access_time, access_time_format, attributes, follow, force, group, mode, modification_time, modification_time_format, owner, path, recurse, selevel, serole, setype, seuser, src, state, unsafe_writes (attr, dest, name)."}</p><p><br></p><p>PLAY RECAP **********************************************************************************************************************************************************</p><p>alpine1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=1&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>centos1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=1&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>redhat1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=1&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>ubuntu1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=1&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p><br></p><p><br></p><p><span style="background-color: rgb(44, 48, 50); color: rgb(222, 219, 214); --darkreader-inline-bgcolor: #232628; --darkreader-inline-color: #d1cdc7;" data-darkreader-inline-bgcolor="" data-darkreader-inline-color="">symbolic_mode is not a valid parameter hence its not a valid bug.</span></p>', 'level': '4', 'id': '42563'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>We do not have a EC2 cluster hence this ansible will not get valid bug</p>', 'level': '2', 'id': '29911'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p>PLAY [NFS Mount Issue Playbook] *************************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] **********************************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>ok: [ubuntu1]</p><p>[WARNING]: Platform linux on host alpine1 is using the discovered Python interpreter at /usr/bin/python3.11, but future installation of another Python interpreter</p><p>could change the meaning of that path. See https://docs.ansible.com/ansible-core/2.14/reference_appendices/interpreter_discovery.html for more information.</p><p>ok: [alpine1]</p><p>ok: [redhat1]</p><p>ok: [centos1]</p><p><br></p><p>PLAY RECAP **********************************************************************************************************************************************************</p><p>alpine1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=0&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>centos1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=0&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>redhat1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=0&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>ubuntu1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=0&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p><br></p><p><br></p><p>Does not reproduce the bug</p>', 'level': '3', 'id': '29626'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p>PLAY [NFS Mount Test] ***********************************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] **********************************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>ok: [ubuntu1]</p><p>[WARNING]: Platform linux on host alpine1 is using the discovered Python interpreter at /usr/bin/python3.11, but future installation of another Python interpreter</p><p>could change the meaning of that path. See https://docs.ansible.com/ansible-core/2.14/reference_appendices/interpreter_discovery.html for more information.</p><p>ok: [alpine1]</p><p>ok: [redhat1]</p><p>ok: [centos1]</p><p><br></p><p>TASK [Create mount point directory] *********************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>changed: [ubuntu1]</p><p>changed: [alpine1]</p><p>changed: [redhat1]</p><p>changed: [centos1]</p><p><br></p><p>TASK [Mount NFS volume] *********************************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>fatal: [ubuntu1]: FAILED! =&gt; {"changed": false, "msg": "missing required arguments: state"}</p><p>fatal: [alpine1]: FAILED! =&gt; {"changed": false, "msg": "missing required arguments: state"}</p><p>fatal: [centos1]: FAILED! =&gt; {"changed": false, "msg": "missing required arguments: state"}</p><p>fatal: [redhat1]: FAILED! =&gt; {"changed": false, "msg": "missing required arguments: state"}</p><p><br></p><p>PLAY RECAP **********************************************************************************************************************************************************</p><p>alpine1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=1&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>centos1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=1&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>redhat1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=1&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>ubuntu1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=1&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p><br></p><p><br></p><p>Missing an argument is not producing valid bug</p>', 'level': '4', 'id': '29626'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p>PLAY [NFS Mount Issue Playbook] *************************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] **********************************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>ok: [ubuntu1]</p><p>[WARNING]: Platform linux on host alpine1 is using the discovered Python interpreter at /usr/bin/python3.11, but future installation of another Python interpreter</p><p>could change the meaning of that path. See https://docs.ansible.com/ansible-core/2.14/reference_appendices/interpreter_discovery.html for more information.</p><p>ok: [alpine1]</p><p>ok: [redhat1]</p><p>ok: [centos1]</p><p><br></p><p>PLAY RECAP **********************************************************************************************************************************************************</p><p>alpine1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=0&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>centos1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=0&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>redhat1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=0&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>ubuntu1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=0&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p><br></p><p><br></p><p>Does not reproduce the bug</p>', 'level': '3', 'id': '29626'}
{'relevance': '1', 'generate_bug': '0', 'comment': '<p>PLAY [NFS Mount Test] ***********************************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] **********************************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>ok: [ubuntu1]</p><p>[WARNING]: Platform linux on host alpine1 is using the discovered Python interpreter at /usr/bin/python3.11, but future installation of another Python interpreter</p><p>could change the meaning of that path. See https://docs.ansible.com/ansible-core/2.14/reference_appendices/interpreter_discovery.html for more information.</p><p>ok: [alpine1]</p><p>ok: [redhat1]</p><p>ok: [centos1]</p><p><br></p><p>TASK [Create mount point directory] *********************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>changed: [ubuntu1]</p><p>changed: [alpine1]</p><p>changed: [redhat1]</p><p>changed: [centos1]</p><p><br></p><p>TASK [Mount NFS volume] *********************************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>fatal: [ubuntu1]: FAILED! =&gt; {"changed": false, "msg": "missing required arguments: state"}</p><p>fatal: [alpine1]: FAILED! =&gt; {"changed": false, "msg": "missing required arguments: state"}</p><p>fatal: [centos1]: FAILED! =&gt; {"changed": false, "msg": "missing required arguments: state"}</p><p>fatal: [redhat1]: FAILED! =&gt; {"changed": false, "msg": "missing required arguments: state"}</p><p><br></p><p>PLAY RECAP **********************************************************************************************************************************************************</p><p>alpine1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=1&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>centos1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=1&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>redhat1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=1&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>ubuntu1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=2&nbsp;&nbsp;changed=1&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=0&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p><br></p><p><br></p><p>Missing an argument is not producing valid bug</p>', 'level': '4', 'id': '29626'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>Does not produce relevant code</p>', 'level': '2', 'id': '71506'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>calling undefined function which are ignored.</p>', 'level': '2', 'id': '12403'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>Non relevant bug</p>', 'level': '3', 'id': '12403'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>PLAY [Redis fact caching bug playbook] ******************************************************************************************************************************</p><p><br></p><p>TASK [Gathering Facts] **********************************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: Platform linux on host alpine1 is using the discovered Python interpreter at /usr/bin/python3.11, but future installation of another Python interpreter</p><p>could change the meaning of that path. See https://docs.ansible.com/ansible-core/2.14/reference_appendices/interpreter_discovery.html for more information.</p><p>ok: [alpine1]</p><p>ok: [redhat1]</p><p>ok: [ubuntu1]</p><p>ok: [centos1]</p><p><br></p><p>TASK [Install epel-release on RHEL6] ********************************************************************************************************************************</p><p>skipping: [ubuntu1]</p><p>skipping: [alpine1]</p><p>skipping: [centos1]</p><p>skipping: [redhat1]</p><p><br></p><p>TASK [Install Redis on RHEL6] ***************************************************************************************************************************************</p><p>skipping: [ubuntu1]</p><p>skipping: [alpine1]</p><p>skipping: [centos1]</p><p>skipping: [redhat1]</p><p><br></p><p>TASK [Enable Redis service] *****************************************************************************************************************************************</p><p>[WARNING]: sftp transfer mechanism failed on [alpine1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [redhat1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>[WARNING]: sftp transfer mechanism failed on [centos1]. Use ANSIBLE_DEBUG=1 to see detailed information</p><p>fatal: [ubuntu1]: FAILED! =&gt; {"changed": false, "cmd": "/usr/bin/systemctl", "msg": "System has not been booted with systemd as init system (PID 1). Can\'t operate.\\nFailed to connect to bus: Host is down", "rc": 1, "stderr": "System has not been booted with systemd as init system (PID 1). Can\'t operate.\\nFailed to connect to bus: Host is down\\n", "stderr_lines": ["System has not been booted with systemd as init system (PID 1). Can\'t operate.", "Failed to connect to bus: Host is down"], "stdout": "", "stdout_lines": []}</p><p>fatal: [alpine1]: FAILED! =&gt; {"changed": false, "msg": "Failed to find required executable \\"systemctl\\" in paths: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"}</p><p>fatal: [redhat1]: FAILED! =&gt; {"changed": false, "msg": "Unable to enable service redis: Failed to enable unit, unit redis.service does not exist.\\n"}</p><p>fatal: [centos1]: FAILED! =&gt; {"changed": false, "msg": "Unable to enable service redis: Failed to enable unit, unit redis.service does not exist.\\n"}</p><p><br></p><p>PLAY RECAP **********************************************************************************************************************************************************</p><p>alpine1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=2&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>centos1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=2&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>redhat1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=2&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p>ubuntu1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: ok=1&nbsp;&nbsp;changed=0&nbsp;&nbsp;unreachable=0&nbsp;&nbsp;failed=1&nbsp;&nbsp;skipped=2&nbsp;&nbsp;rescued=0&nbsp;&nbsp;ignored=0</p><p><br></p>', 'level': '3', 'id': '13703'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p><span style="background-color: rgb(57, 62, 64); color: rgb(206, 202, 195); --darkreader-inline-bgcolor: #2e3133; --darkreader-inline-color: #c6c1b9;" data-darkreader-inline-bgcolor="" data-darkreader-inline-color="">The </span><code style="background-color: rgb(57, 62, 64); color: var(--darkreader-text--tw-prose-code); --darkreader-inline-bgcolor: #2e3133; --darkreader-inline-color: var(--darkreader-text--darkreader-text--tw-prose-code);" data-darkreader-inline-bgcolor="" data-darkreader-inline-color="">lookup</code><span style="background-color: rgb(57, 62, 64); color: rgb(206, 202, 195); --darkreader-inline-bgcolor: #2e3133; --darkreader-inline-color: #c6c1b9;" data-darkreader-inline-bgcolor="" data-darkreader-inline-color=""> function expects two arguments. In the expression </span><code style="background-color: rgb(57, 62, 64); color: var(--darkreader-text--tw-prose-code); --darkreader-inline-bgcolor: #2e3133; --darkreader-inline-color: var(--darkreader-text--darkreader-text--tw-prose-code);" data-darkreader-inline-bgcolor="" data-darkreader-inline-color="">\'INVENTORY\' in lookup(\'env\', \'\')</code><span style="background-color: rgb(57, 62, 64); color: rgb(206, 202, 195); --darkreader-inline-bgcolor: #2e3133; --darkreader-inline-color: #c6c1b9;" data-darkreader-inline-bgcolor="" data-darkreader-inline-color="">, it appears that you are passing an empty string </span><code style="background-color: rgb(57, 62, 64); color: var(--darkreader-text--tw-prose-code); --darkreader-inline-bgcolor: #2e3133; --darkreader-inline-color: var(--darkreader-text--darkreader-text--tw-prose-code);" data-darkreader-inline-bgcolor="" data-darkreader-inline-color="">\'\'</code><span style="background-color: rgb(57, 62, 64); color: rgb(206, 202, 195); --darkreader-inline-bgcolor: #2e3133; --darkreader-inline-color: #c6c1b9;" data-darkreader-inline-bgcolor="" data-darkreader-inline-color=""> as the second argument to the </span><code style="background-color: rgb(57, 62, 64); color: var(--darkreader-text--tw-prose-code); --darkreader-inline-bgcolor: #2e3133; --darkreader-inline-color: var(--darkreader-text--darkreader-text--tw-prose-code);" data-darkreader-inline-bgcolor="" data-darkreader-inline-color="">lookup</code><span style="background-color: rgb(57, 62, 64); color: rgb(206, 202, 195); --darkreader-inline-bgcolor: #2e3133; --darkreader-inline-color: #c6c1b9;" data-darkreader-inline-bgcolor="" data-darkreader-inline-color=""> function. This could be the reason for the </span><code style="background-color: rgb(57, 62, 64); color: var(--darkreader-text--tw-prose-code); --darkreader-inline-bgcolor: #2e3133; --darkreader-inline-color: var(--darkreader-text--darkreader-text--tw-prose-code);" data-darkreader-inline-bgcolor="" data-darkreader-inline-color="">IndexError: list index out of range</code><span style="background-color: rgb(57, 62, 64); color: rgb(206, 202, 195); --darkreader-inline-bgcolor: #2e3133; --darkreader-inline-color: #c6c1b9;" data-darkreader-inline-bgcolor="" data-darkreader-inline-color=""> error, since the </span><code style="background-color: rgb(57, 62, 64); color: var(--darkreader-text--tw-prose-code); --darkreader-inline-bgcolor: #2e3133; --darkreader-inline-color: var(--darkreader-text--darkreader-text--tw-prose-code);" data-darkreader-inline-bgcolor="" data-darkreader-inline-color="">env</code><span style="background-color: rgb(57, 62, 64); color: rgb(206, 202, 195); --darkreader-inline-bgcolor: #2e3133; --darkreader-inline-color: #c6c1b9;" data-darkreader-inline-bgcolor="" data-darkreader-inline-color=""> plugin is trying to look up an environment variable with an empty name.</span></p>', 'level': '3', 'id': '40082'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>None</p>', 'level': '4', 'id': '40082'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>We do not have a host with full disk</p>', 'level': '3', 'id': '73179'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>We do not have a full disk</p>', 'level': '4', 'id': '73179'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>Does not produce related code</p>', 'level': '3', 'id': '13876'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>generates buggy code.</p>', 'level': '4', 'id': '13876'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>invalid filepath causes the error. </p>', 'level': '2', 'id': '30821'}
{'relevance': '0', 'generate_bug': '0', 'comment': '<p>Invalid filepath causes error. not valid bug.</p>', 'level': '3', 'id': '30821'}
